pkg/products/bamboo/helm_release.tf

```diff
@@ -59,4 +59,34 @@ data "kubernetes_service" "bamboo" {
     name      = "bamboo"
     namespace = kubernetes_namespace.bamboo.metadata[0].name
   }
+}
+
+resource "helm_release" "bamboo_agent" {
+  name       = "bamboo-agent"
+  namespace  = kubernetes_namespace.bamboo.metadata[0].name
+  repository = "https://atlassian.github.io/data-center-helm-charts"
+  chart      = "bamboo-agent"
+  version    = "0.0.2"
+
+  depends_on = [helm_release.bamboo]
+
+  values = [
+    yamlencode({
+      replicaCount = var.number_of_agents
```

Comment:
How does this work if it is 0?

Is defining `number_of_agents = 0` a valid way how to disable agents?

Reply:
yes, `replicaCount = 0` will disable agents. i.e. the helm release will be installed, but no agent pod will be initiated. 

I was thinking about not install agent chart at all when this variable is set to 0. But had a rethink, installing Helm release with replicaCount = 0 will actually save user some time preparing agent's values.yaml file. To deploy agents later, user can just run kubectl command to scale up the deployments.

Reply:
It worth to point to this in documentation. (`CONFIGURATION.md`  and/or `config.tfvars`)?